site/zenodo_rdm/custom_fields/domain_fields.py

```diff
@@ -167,381 +90,380 @@ def field(self):
             ),
         ),
         dict(
             field="dwc:typeStatus",
+            ui_widget="MultiInput",
             props=dict(
-                label="Type status",
+                label=_("Type status"),
             ),
         ),
         dict(
             field="dwc:verbatimElevation",
+            ui_widget="MultiInput",
             props=dict(
-                label="Verbatim elevation",
+                label=_("Verbatim elevation"),
             ),
         ),
         dict(
             field="dwc:verbatimEventDate",
+            ui_widget="MultiInput",
             props=dict(
-                label="Verbatim event date",
+                label=_("Verbatim event date"),
             ),
         ),
         # gbif-dwc
         dict(
             field="gbif-dwc:recordedByID",
+            ui_widget="MultiInput",
             props=dict(
-                label="Recorded by ID",
+                label=_("Recorded by ID"),
             ),
         ),
         dict(
             field="gbif-dwc:identifiedByID",
+            ui_widget="MultiInput",
             props=dict(
-                label="Identified by ID",
-            ),
-        ),
-        dict(
-            field="ac:associatedSpecimenReference",
-            props=dict(
-                label="Associated specimen reference",
-            ),
-        ),
-        dict(
-            field="ac:physicalSetting",
-            props=dict(
-                label="Physical setting",
+                label=_("Identified by ID"),
             ),
         ),
+        # openbiodiv
         dict(
-            field="ac:associatedSpecimenReference",
+            field="openbiodiv:TaxonomicConceptLabel",
+            ui_widget="MultiInput",
             props=dict(
-                label="Associated specimen reference",
+                label=_("Taxonomic concept label"),
             ),
         ),
         dict(
-            field="ac:captureDevice",
+            field="dc:creator",
+            ui_widget="MultiInput",
             props=dict(
-                label="Capture device",
+                label=_("Creator"),
             ),
         ),
         dict(
-            field="ac:resourceCreationTechnique",
+            field="dc:rightsHolder",
+            ui_widget="MultiInput",
             props=dict(
-                label="Resource creation technique",
+                label=_("Rights holder"),
             ),
         ),
+        # obo
         dict(
-            field="ac:subjectOrientation",
+            field="obo:RO_0002453",
+            template="zenodo_rdm/obo.html",
+            ui_widget="MultiInput",
             props=dict(
-                label="Subject orientation",
+                label="Host of",
             ),
         ),
+    ],
+}
+
+AUDIOVIS_FIELDS_UI = {
+    "section": _("Audiovisual core"),
+    "icon": "camera",
+    "hide_from_upload_form": False,
+    "compose_fields": True,  # marks the group as dynamic
```

Comment:
We need a better naming for this :) Not sure what yet.....

Reply:
what about be more explicit and say something like 
- `show_all_fields_by_default`
- `show_fields_on_demand`
- `search_fields` (kind of dropdown paradigm of show options vs search for options)